sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_daily_v2/metadata.yaml

```diff
@@ -8,30 +8,28 @@ description: |
   along with some attributes related to that user.
   See the schema for a list of attributes and their descriptions.
 
-  Partitioned by submission_date,
-  clustered by country, os_name, and user_id
-
   This lower level model uses the following category of events:
   event_category IN ('auth', 'auth_bounce', 'content', 'oauth')
 
   Partitioned by submission_date,
-  clustered by country, os_name, and user_id
+  clustered by registered, seen_in_tier1_country, country, os_name
 owners:
   - kik@mozilla.com
 labels:
   application: fxa
   incremental: true
   schedule: daily
-# TODO: once we agree on the model scheduling needs to be uncommented.
-# scheduling:
-#   dag_name: bqetl_fxa_events
+scheduling:
+  dag_name: bqetl_fxa_events
+  start_date: "2024-01-26"
 bigquery:
   time_partitioning:
     type: day
     field: submission_date
     require_partition_filter: true
   clustering:
     fields:
+      - registered
+      - seen_in_tier1_country
       - country
       - os_name
```

Comment:
I haven't changed clustering on a BQ table recently; is that something that artifact deployment will automically handle, or do you need to take some manual action to put that change into place?

Reply:
We will need to redeploy the table, but in this case it's not an issue as the table is empty anyways.

Reply:
All three tables have been recreated.